Mathlib/Analysis/NormedSpace/Banach.lean

```diff
@@ -353,6 +353,27 @@ theorem coeFn_toContinuousLinearEquivOfContinuous_symm (e : E ≃ₗ[𝕜] F) (h
 
 end LinearEquiv
 
+namespace ContinuousLinearMap
+
+variable [CompleteSpace E]
+
+/-- An injective continuous linear map with a closed range defines a continuous linear equivalence
+between its domain and its range. -/
+noncomputable def toContinuousLinearEquivOfInjectiveOfIsClosed
```

Comment:
Given the other existing things in the library, I think it makes sense to call this `ContinuousLinearMap.rangeEquiv`. Don't forget to rename the lemma too, and shuffle arguments to new lines for style so the spacing isn't weird.
```suggestion
noncomputable def rangeEquiv
```